godot-google-play-billing/build.gradle

```diff
@@ -9,14 +9,9 @@ android {
     defaultConfig {
         minSdkVersion 18
         targetSdkVersion 29
+        versionName '1.0.0'
+        archivesBaseName = "GodotGooglePlayBilling.$versionName"
     }
-
-    libraryVariants.all { variant ->
-        variant.outputs.all { output ->
-            output.outputFileName = "GodotGooglePlayBilling.${variant.name}.aar"
-        }
-    }
```

Comment:
@timoschwarzer If you want to have the `.` instead of `-`, you have to use this logic for the name. You can include the version name by storing it as a variable and using the variable here and in `defaultConfig`.

Reply:
Thanks!